src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        int supply = 0;
+        int buy = 0;
+        int result;
+        String report = null;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
```

Comment:
try to not do everything in single method, lets use such pattern:
String data = readFromFile(file);
String report = createReport(data);
writeToFile(file, report)